### PR TITLE
Spsh 641

### PR DESCRIPTION
**Pull request by Nael-Alshowaikh**
# Description

## Approval for review
- [ ] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [ ] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

### CODE REVIEW

File: compose.yaml [hunk 1]
```diff
@@ -106,3 +106,17 @@ services:
...
+    ports:
+      - '4317:4317'
+      - '4318:4318'
+    volumes:
+      - ./otel-config.yaml:/etc/otel-config.yaml
+    command: ['--config=/etc/otel-config.yaml']
+
+networks:
+  default:
+    external:
```
Comment (DPDS93CT) [inline compose.yaml:121]: At least with Docker Compose V. 2.21 this configuration does not work, what version is used?

File: otel-config.yaml
Comment (DPDS93CT) [inline otel-config.yaml:1]: This may also be moved to /config folder. For the cluster I put it here /charts/dbildungs-iam-server/config/telemetry

**Review by kristoff-kiefer — CHANGES_REQUESTED** [review]
Move the otel config to config please

File: charts/dbildungs-iam-server/templates/telemetry-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      app.kubernetes.io/name: {{ template "common.names.name" . }}
+      app.kubernetes.io/component: server-telemetry
+  replicas: {{ .Values.replicaCount }}
+  template:
+    metadata:
+      labels:
+        {{- include "common.labels" . | nindent 8 }}
+        app.kubernetes.io/component: server-telemetry
+    spec:
+      containers:
```
Comment (kristoff-kiefer) [inline charts/dbildungs-iam-server/templates/telemetry-deployment.yaml:21]: I strongly doubt we need that. The collection should be done by SRE. Just providing the endpoints in the main app should be enough in the cluster. Just add a service that makes them publicly available

**Review by kristoff-kiefer — CHANGES_REQUESTED** [review]
Collection is done by SRE

File: otel-config.yaml
Comment (DPDS93CT) [reply otel-config.yaml:1]: done

**Pull request closed without merging**